### PR TITLE
chore: disable timing and hash by default for stats.toString()

### PR DESCRIPTION
**Pull request by hardfist**
## Summary
I brings extra burden for outside contributors to disable timing and hash for stat snapshot and which not aligns with webpack behaviors too.
so we align this behavior with webpack(disable hash and timing by default in string snapshot)

## Require Documentation?

- [x] No
- [ ] Yes, the corresponding rspack-website PR is \_\_

### CODE REVIEW

File: packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap [hunk 1]
```diff
@@ -1339,6 +1340,7 @@ exports[`StatsTestCases should print correct stats for limit-chunk-count-plugin
       "outputPath": "<PROJECT_ROOT>/tests/statsCases/limit-chunk-count-plugin/dist",
       "publicPath": "",
       "rspackVersion": "0.3.8",
+      "time": 64,
```
Comment (ahabhgk) [inline packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap:1343]: Remove this

File: packages/rspack/tests/statsCases/limit-chunk-count-plugin/webpack.config.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ module.exports = [1, 2, 3, 4].map(n => ({
 		})
 	],
 	stats: {
-		hash: false,
-		timings: false,
```
Comment (ahabhgk) [inline packages/rspack/tests/statsCases/limit-chunk-count-plugin/webpack.config.js:19]: Add this back to keep snapshot stable

File: packages/rspack/package.json [hunk 3]
```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "tsc -b ./tsconfig.build.json",
     "dev": "tsc -w",
-    "test": "cross-env NO_COLOR=1 RSPACK_DEP_WARNINGS=false node --expose-gc --max-old-space-size=8192 --experimental-vm-modules ../../node_modules/jest-cli/bin/jest --runInBand --logHeapUsage"
+    "test": "cross-env NO_COLOR=1 RSPACK_DEP_WARNINGS=false node --expose-gc --max-old-space-size=8192 --experimental-vm-modules ../../node_modules/jest-cli/bin/jest --runInBand --logHeapUsage --updateSnapshot"
```
Comment (ahabhgk) [inline packages/rspack/package.json:18]: I suggest adding a new script to update instead of changing this

Comment (hardfist) [reply packages/rspack/package.json:18]: forget to revert it back

File: packages/rspack/tests/statsCases/limit-chunk-count-plugin/webpack.config.js (hunk 2 above)
Comment (hardfist) [reply packages/rspack/tests/statsCases/limit-chunk-count-plugin/webpack.config.js:19]: this is not included in original webpack tests, so we may don't need this

File: packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap (hunk 1 above)
Comment (hardfist) [reply packages/rspack/tests/__snapshots__/StatsTestCases.test.ts.snap:1343]: I will check out the default behavior of webpack

File: packages/rspack/package.json (hunk 3 above)
Comment (hardfist) [reply packages/rspack/package.json]: reverted

**Review by ahabhgk — DISMISSED** [review]

**Pull request closed without merging**